src/Altinn.App.Api/Controllers/DataController.cs

```diff
@@ -289,7 +291,7 @@ private static bool FileValidationEnabledForDataType(DataType dataTypeFromMetada
                 }
                 else if (dataType.AppLogic?.ClassRef is not null)
                 {
-                    return await GetFormData(org, app, instanceOwnerPartyId, instanceGuid, dataGuid, dataType, language, instance);
+                    return await GetFormData(org, app, instanceOwnerPartyId, instanceGuid, instance, dataGuid, dataElement, dataType, includeRowId, language);
```

Comment:
Do we need both instance and instanceGuid?

Reply:
It is just that `Guid.Parse(instance.Id.Split("/")[1])` gets a little repetitive.